### ISSUE TITLE
[ic-side-navigation]: Screen reader announces side navigation menu items twice when initially crawling through page.

### ISSUE THREAD

**Issue by jd239** [mi6/ic-ui-kit#2642]
## Summary of the bug 
When initially loading a page (which has a side navigation) with Mac VoiceOver, the screen reader cursor will highlight each menu item and announce the menu item twice.

## 🪜 How to reproduce 
Tell us the steps to reproduce the problem: 
1. Go to page: https://design.sis.gov.uk/side-navigation-pattern/ with Mac VO on. (You may need to refresh the page). 
2. Watch as the screen reader cursor goes through the side navigation reading out the menu items twice. 

## 📸 Screenshots or code 
N/A

## 🖥 📱 Device 
- Type: Desktop
- Device: Macbook
- OS version: Sonoma 14.6.1
- Browser version: Chrome 130

## 🧐 Expected behaviour 
Screen reader should announce each menu item once

## 📝 Acceptance Criteria  
If relevant, describe in full detail the different interactions and edge cases that the component or patterns needs to fulfil. 

*Given that I have loaded up a page with the side navigation and the screen reader is enabled* 
*When the screen reader cursor is scanning the side navigation component*  
*Then each menu item should be announced once* 

## Additional info 
After testing on the expanded side navigation, the issue may be the tooltip which is present when collapsed also being read out by the screen reader.

**Comment by GCHQ-Developer-299** [comment]
Multiple developers unable to replicate in Voiceover when tested today - assuming we fixed this in the interim and closing ticket :)
